### PR TITLE
refactor(templates/use-cases): migrate to Chakra

### PR DESCRIPTION
**Pull request by TylerAPfledderer**
## Description
Migrate the `use-cases` template to Chakra UI

## Related Issue
#9353

### CODE REVIEW

File: src/templates/use-cases.tsx [hunk 1]
```diff
@@ -25,135 +39,66 @@ import Logo from "../components/Logo"
...
+import UpgradeTableOfContents from "../components/UpgradeTableOfContents"
+import TableOfContents from "../components/TableOfContents"
 import Translation from "../components/Translation"
 import SectionNav from "../components/SectionNav"
 import {
   Divider,
   Paragraph,
-  Header1,
-  Header4,
   ListItem,
```
Comment (pettinarip) [inline src/templates/use-cases.tsx:50]: We need to deprecate these components as well.

- Paragraph => `Text`
- ListItem => `<chakra.li>`

File: src/templates/use-cases.tsx [hunk 2]
```diff
@@ -25,135 +39,66 @@ import Logo from "../components/Logo"
...
-  h1: Header1,
+  h1: H1,
   h2: H2,
   h3: H3,
-  h4: Header4,
+  h4: H4,
   p: Paragraph,
   li: ListItem,
-  pre: Pre,
+  // pre: Pre,
```
Comment (pettinarip) [inline src/templates/use-cases.tsx:101]: Isn't this used?

File: src/templates/use-cases.tsx [hunk 3]
```diff
@@ -173,134 +118,145 @@ const components = {
...
-    display: none;
-  }
-`
+const HeroContainer = (props: ChildOnlyProp) => (
+  <Flex
+    bg="cardGradient"
+    boxShadow="inset 0px -1px 0px rgba(0, 0, 0, 0.1)"
+    flexDirection={{ base: "column-reverse", lg: "row" }}
+    justifyContent="flex-end"
+    minH="608px"
```
Comment (pettinarip) [inline src/templates/use-cases.tsx:127]: I think we need to force this height. In prod we have this defined height:
image


```suggestion
    h="608px"
```

**Review by pettinarip — CHANGES_REQUESTED** [review]
gj @TylerAPfledderer I've pointed only a few things.

For example, the page Hero, has a height much bigger than in prod. Take a look at the defi page:
https://ethereumorgwebsitedev01-tylerapfleddererethereumo62717.gatsbyjs.io/en/defi/
vs
https://ethereum.org/en/defi/

File: src/templates/use-cases.tsx (hunk 2 above)
Comment (TylerAPfledderer) [reply src/templates/use-cases.tsx:101]: Whoops! Not sure what I did here. Adding it back in. 😅

**Comment by TylerAPfledderer** [comment]
@pettinarip this should be good to go!

**Review by pettinarip — APPROVED** [review]
lgtm! 💯

**Pull request merged**